R/convertTRACCS.R

```diff
@@ -0,0 +1,19 @@
+#' Convert TRACCS road transportation data to iso country.
+#'
+#' @param subtype One of the possible subtypes, see default argument.
+#' @return magclass object
+#'
+#' @examples
+#' \dontrun{
+#' a <- readSource("TRACCS")
+#' }
+#' @author Alois Dirnaichner
+#' @seealso \code{\link{readSource}}
+#' @importFrom madrat toolCountryFill getIsoList toolCountry2isocode
+convertTRACCS <- function(x, subtype) {
+  getItems(x, dim=1) <- toolCountry2isocode(getItems(x, dim=1))
+  getSets(x)["d1.1"] <- "iso"
+  x <- toolCountryFill(x, fill = NA, countrylist = getISOlist(), verbosity = 2)
```

Comment:
Setting `countrylist = getISOlist()` might be redundant

Reply:
as far as I can tell, `countrylist=NULL` in https://github.com/pik-piam/madrat/blob/master/R/toolCountryFill.R#L43

Reply:
```suggestion
  x <- toolCountryFill(x, fill = NA, verbosity = 2)
```